enrollment-server/src/main/java/com/wultra/app/enrollmentserver/controller/api/MobileTokenController.java

```diff
@@ -106,14 +158,20 @@ public Response operationApprove(@RequestBody ObjectRequest<OperationApproveRequ
                 throw new MobileTokenAuthException();
             }
 
+            final String operationId = requestObject.getId();
+            final String data = requestObject.getData();
+            if (operationId == null) {
+                throw new MobileTokenAuthException();
```

Comment:
We should probably log the error reason, otherwise this will result in a very generic log message.